src/pages/Main/PersonProfile/index.tsx

```diff
@@ -145,14 +159,18 @@ const PersonView = React.memo(function ({ persons, person }) {
     );
 });
 
-const PersonSelectionView = React.memo(function ({ persons }) {
+const PersonSelectionView = React.memo(function ({
+    persons,
+}: {
+    persons: PersonsDict;
+}) {
     const classes = useStyles();
     const navigate = useNavigate();
 
-    const rands = new Set();
+    const rands = new Set<string>();
     const ids = Object.keys(persons);
     while (rands.size < 5 && rands.size < ids.length) {
-        const index = parseInt(Math.random() * ids.length);
+        const index = Math.random() * ids.length;
```

Comment:
Math.random() returns float, int * float should return another float, how would you use it as an index?

Reply:
you are right i chaged it to Math.floor(). 
parseInt is not right is this case